srcpkgs/atuin/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'atuin'
+pkgname=atuin
+version=12.0.0
+revision=1
+build_style=cargo
+build_helper=qemu
+hostmakedepends="pkg-config"
+makedepends="libsodium-devel sqlite-devel"
```

Comment:
https://github.com/launchbadge/sqlx/issues/191

Reply:
and also related: https://github.com/rusqlite/rusqlite/issues/769

Reply:
thank you for the heads-up!

Reply:
Less of a heads up and more of a merge blocker sadly.. What we need to do is maintaining an sqlx fork somewhere that changes this, and then maintaining patch files that use https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section to override where it fetches sqlx from, with potentially multiple branches for the different sqlx versions... Unless upstream "fixes" this allowing us to override this in an easier way.

Reply:
What stops us from shipping the statically-linked version for the time being?

Reply:
Pretty sure that's just what Void does. I've had to put in quite a bit of work for things like nushell not linking sqlite statically for example.

Reply:
Yeah, so my understanding is, as long as the package builds and runs, we could merge this for now as-is, no?

Reply:
I don't think so :/

I've been using this as-is for a while and would be happy if we could get it in, but I think it might be necessary to do something similar to what we're doing in nushell to get sqlite to not be linked statically.

Reply:
@tranzystorek-io can you try getting the sqlite dynamically linked? Shouldn't be too tricky to do that, and should make this more likely to be merged.

Reply:
This is resolved now, right?

Reply:
I don't think so, atuin still refuses to dynlink sqlite, and I don't have enough spare time to do much about it. The other option would be to accept the statically built and linked version as-is now.